### PR TITLE
BZ#1945244 Note hybrid networking requirement for WINC

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1945244

Preview: https://deploy-preview-31317--osdocs.netlify.app/openshift-enterprise/latest/installing/installing-preparing.html#installing-preparing-cluster-for-workloads

### CODE REVIEW

**Comment by codyhoag** [comment]
@sgaoshang PTAL

File: installing/installing-preparing.adoc [hunk 1]
```diff
@@ -85,7 +85,7 @@ For a production cluster, you must configure the following integrations:
...
+Depending on your workload needs, you might need to take extra steps before you begin deploying applications. For example, after you prepare infrastructure to support your application xref:../cicd/builds/build-strategies.adoc#build-strategies[build strategy], you might need to make provisions for xref:../scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc#cnf-performance-addon-operator-for-low-latency-nodes[low-latency] workloads or to xref:../nodes/pods/nodes-pods-secrets.adoc#nodes-pods-secrets[protect sensitive workloads]. You can also configure xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[monitoring] for application workloads. If you plan to run xref:../windows_containers/enabling-windows-container-workloads.adoc#enabling-windows-container-workloads[Windows workloads], you must enable xref:../networking/ovn_kubernetes_network_provider/configuring-hybrid-networking.adoc#configuring-hybrid-networking[hybrid networking with OVN-Kubernetes] during the installation process; hybrid networking cannot be enabled after your cluster is installed.
```
Comment (jboxman) [inline installing/installing-preparing.adoc:88]: @codyhoag, so I wonder if this includes enough items that a bullet list is more easily scannable?

**Review by jboxman — APPROVED** [review]
One suggestion; LGTM.

File: installing/installing-preparing.adoc (hunk 1 above)
Comment (codyhoag) [reply installing/installing-preparing.adoc:88]: Good point; I'm going to make a note of that for a follow-up, but merge the current change as-is. Thanks!

**Pull request merged**